JoyboyCommunity/src/components/TippingModal/index.tsx

```diff
@@ -0,0 +1,58 @@
+import {useNavigation} from '@react-navigation/native';
+import React from 'react';
+import {View} from 'react-native';
+import {Portal} from 'react-native-portalize';
+
+import {TippingModalIcon} from '../../assets/icons';
+import {useStyles} from '../../hooks';
+import {MainStackNavigationProps} from '../../types';
+import {Button, Text} from '..';
+import stylesheet from './styles';
+
+export type TippingModalProps = {
+  user: string;
+  amount: number;
+  visible?: boolean;
+};
+
+export const TippingModal: React.FC<TippingModalProps> = ({user, amount, visible = true}) => {
+  const styles = useStyles(stylesheet);
+
+  const navigation = useNavigation<MainStackNavigationProps>();
+
+  if (!visible) return null;
+
+  return (
+    <Portal>
+      <View style={styles.container}>
+        <View style={styles.modal}>
+          <View style={styles.icon}>
+            <TippingModalIcon />
+          </View>
+
+          <View style={styles.content}>
+            <Text color="#14142C" weight="bold" fontSize={21} lineHeight={24}>
```

Comment:
Colors should be the name of the color, these colors are defined in the theme.
- `#14142C` should be `text`
- `#EC796B` should be `primary`
- `#6B6B8C` should be `textSecondary`